CONTRIBUTING.md

```diff
@@ -144,8 +144,9 @@ Two approvals are required for a PR to be merged. A PR can then be merged into t
 
 On merging, you can rework the branch history as you see fit. Consider
 squashing commits from the original PR with those made during the review
-process, for example. A [force push][git-force] will be required if you do
-this.
+process, for example. Also ideally commit messages should follow the format
```

Comment:
(grammar nitpick): I think the sentence should either start `Also, ideally, commit ....` or `Ideally, commit ....`

Reply:
Yeah, I think you're right. But even more nitpicky - having both _ideally_ and _should_ in the same sentence isn't idea. I'd probably just make this `Commit messages should follow...`